browser/brave_browser_main_extra_parts.cc

```diff
@@ -34,5 +56,15 @@ void BraveBrowserMainExtraParts::PreMainMessageLoopRun() {
   // created before browser thread creation.
   // So, call it after browser threads are created.
   g_brave_browser_process->brave_widevine_bundle_manager()->StartupCheck();
+#endif
+  // Disabled on mobile platforms, see for instance issues/6176
+#if !defined(OS_ANDROID)
+  // TODO(iefremov): Maybe find a better place for this initialization.
+  g_brave_browser_process->brave_p3a_service()->Init();
```

Comment:
if p3a doesn't work in android right now then we need a p3a buildflag that is disabled in android. `OS_ANDROID` should be used exclusively for platform-specific issues and not feature/component enable/disable

Reply:
done